src/lib/mdns/Advertiser.h

```diff
@@ -32,10 +32,27 @@ static constexpr uint16_t kMdnsPort = 5353;
 // Need 8 bytes to fit a thread mac.
 static constexpr size_t kMaxMacSize = 8;
 
+#define CHIP_MDNS_KEY_DISCRIMINATOR_MAXLENGTH 5
```

Comment:
Feels like all these defines should be constants like below, we've been moving all constants that way stylistically